process/request_resources.md

```diff
@@ -0,0 +1,60 @@
+
+To receive space in an OpenSSF newsletter, projects should contact the Marketing Committee a minimum of two weeks before the newsletter publication date. Projects are responsible for tracking due dates and submitting materials.
+Technical content should be reviewed by the TAC, and potentially other WG's or Committees as appropriate. This includes, but is not limited to: white papers, technical briefings, policy recommendations, etc. This does not include project-specific announcements, such as those relating to project milestone releases. 
+
+### Contributor meet ups at OpenSSF events
+
+Projects that would like contributor meet up space at OpenSSF events should contact the OpenSSF Marketing Committee with a request six months before the event date, or as soon as the event is announced if it is less than six months. Late requests may be accommodated but are not guaranteed. The events team handles room reservations, attendee registration if necessary, and connecting the project leads to event and facility support staff. Projects are responsible for organizing and running their own contributor meet up.
+
+### Swag
+
+The OpenSSF Marketing Committee keeps a small, annual swag stipend for all projects. Projects may wish to seek outside funding for project swag beyond this stipend.
+
+## Requests through the OpenSSF Budget Committee
+Requests that go through the OpenSSF Budget Committee are reviewed at regularly occuring Budget Committee meetings. Projects should plan their funding needs and requests with this timeline in mind.
+
+### Infrastructure
+
+_Program TBD as of June 2022. Intended program details below._
+Central infrastructure is available to qualifying projects for test and build support. Projects should submit requests on an annual basis to the OpenSSF Budget Committee, a minimum of three months before the requested start date. Projects are responsible for submitting a resource forecast with their request. The Budget Committee and project will work together to agree on minimum infrastructure support based on annual budget. Projects are responsible for communicating to the Budget Committee if there are anticipated increases (including spike events) arise. 
+
+### Maintainer stipends
+
+_Program TBD as of June 2022. Intended program details below._
+Limited, time-bound maintainer stipends for specified project improvements may be available to qualifying projects. Projects should submit a request to the OpenSSF Budget Committee with proposed scope, cost, desired outcome, TAC sponsor approval, and timeline at least three months before the requested start date. Maintainer stipends may not be paid to employees of OpenSSF member companies (excludes Associate members).
```

Comment:
Please drop this. There is contracting work we have paid for, and will pay for, based on identified needs and the like, but the Budget Committee is not involved with that. Let's do a separate PR for this.

Reply:
+1

Reply:
+1